crates/xilem_core/src/view/rc.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 the Xilem Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#[macro_export]
+macro_rules! generate_rc_view {
+    (($($rc_ty:tt)*), $viewtrait:ident, $viewmarker:ty, $cx:ty, $changeflags:ty; $($ss:tt)*) => {
```

Comment:
```suggestion
    ($rc_ty:path, $viewtrait:ident, $viewmarker:ty, $cx:ty, $changeflags:ty; $($ss:tt)*) => {
```

Reply:
Just tried that locally, but it has issues with `$rc_ty<V>` unfortunately (I have tried `ty` previously, same issue):
```
expected one of `!`, `+`, `where`, or `{`
```

Not sure if there's another way to have `<V>` after a path-like that is not `tt`.

Reply:
For `ty` it makes sense to me, since there can't be generics after a type in general (any generics would be part of the type, not separate), for `path` I don't understand it but oh well. You could use `$($rc_ty:ident)::*` or `$($tc_ty:tt)::*` to get rid of the surrounding parentheses though, right?